coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -1196,67 +1196,62 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
     validatePlan(execPlan1, expectedPlan1)
   }
 
-  it ("should pushdown aggregations/binary-joins when leaf plans are split across partitions") {
+  it ("should pushdown when leaf plans are split across partitions") {
     val startSec = 123
-    val stepSec = 456
-    val endSec = 789
-    val queryExpectedPairs = Seq(
-      ("""sum(rate(test{job="app"}[5m]))""",
```

Comment:
Replaced these with non-string-based tests. Since `Aggregate`s now have the same simple pushed-down structure as the others, we can just programmatically check that each query materializes to a 3-node tree with a `MultiPartitionDistConcatExec` root, two `PromqlRemoteExec` children, and all the expected query parameters.